share/completions/mkdir.fish

```diff
@@ -0,0 +1,9 @@
+complete -c mkdir -s m -l mode --description 'Set file mode (as in chmod)' -x
+complete -c mkdir -s p -l parents --description 'Make parent directories as needed'
+complete -c mkdir -s v -l verbose --description 'Print a message for each created directory'
+complete -c mkdir -s Z --description 'Set SELinux security context of each created directory to the default type'
```

Comment:
Only `killall`, `modprobe` and `open` indeed. I'll use `mkdir --version` then.

I guess it's okay if I use `uname` for `modinfo` completions? I have a few other command completions done.